lib/src/query.dart

```diff
@@ -1,5 +1,15 @@
 import 'dart:js_interop';
+import 'dart:js_util';
+
+import 'package:deno_postgres_interop/src/result_type.dart';
 
 /// [deno-postgres@v0.17.0/Query](https://deno.land/x/postgres@v0.17.0/query/query.ts?s=Query).
 @JS()
-class Query<T> {}
+class Query {}
+
+/// [deno-postgres@v0.17.0/Query](https://deno.land/x/postgres@v0.17.0/query/query.ts?s=Query).
+extension QueryProps on Query {
+  /// [deno-postgres@v0.17.0/Query/result_type](https://deno.land/x/postgres@v0.17.0/query/query.ts?s=Query#prop_result_type).
+  ResultType get resultType =>
+      ResultType.values[getProperty(this, 'result_type')];
```

Comment:
not totally safe because the indexes are not defined in JS code, but IDK how to map this value otherwise

This will have to do for now, and I'll investigate it further in the case of any problems